### DISCUSSION TITLE
SEB crash on iPad - solution, but ...

### DISCUSSION THREAD

**Discussion by sammanise** [SafeExamBrowser/seb-mac#167] [Q&A]
SEB does crash in iPadOS15.1 when clicking a BookWidgets-link. This can be solved by restarting the iPad (or do a cold restart by pressing and holding the standby-and-homebuttons), opening the SEB-app and then clicking a BW-link. 
This is workable, but cumbersome. Is there a different / more elegant solution for this or do we have to wait for a much-needed update?

**Comment by danschlet** [comment]
You can start SEB first, switch back to the Home Screen and open the exam link then (don't know what BookWidget is).
This issue will be fixed in the 3.1 release.
